src/main/ngapp/src/app/login/login.component.ts

```diff
@@ -11,6 +11,8 @@ export class LoginComponent implements OnInit {
  loginAwait = false;
  error: any;
  errorPresent = false;
+ passwordNotRight = false; // Tracks if a login was successful
```

Comment:
rename to loginSuccessful = true?

Reply:
alright. 
implemented.

Reply:
did you send out new commits? Can't see one.

Reply:
new commits sent ( i had been waiting for reply to the comment on line 62 before making a commit)